src/nrf5_sdk15_platform/communication/ruuvi_nrf5_sdk15_communication_ble_gatt.c

```diff
@@ -318,6 +321,9 @@ static void ble_evt_handler (ble_evt_t const * p_ble_evt, void * p_context)
             evt.type = BLE_NUS_EVT_COMM_STOPPED;
             nus_data_handler (&evt);
             m_conn_handle = BLE_CONN_HANDLE_INVALID;
+            err_code = ri_gatt_params_request (RI_GATT_STANDARD);
```

Comment:
This is requesting parameter change for invalid change and will error, delete this line

Reply:
Done.